app/models/task.rb

```diff
@@ -33,7 +33,5 @@ def file_no_business!
     end
   end
 
-  def supplier_name
-    Supplier.find_by(id: supplier.id).name
-  end
+  delegate :name, to: :supplier, prefix: true
```

Comment:
Could this change be squashed into the previous commit, as it's not related to adding the User endpoint?